04/app.js

```diff
@@ -5,4 +5,48 @@ const menuItems = [
     {text: 'start', url: '/'},
     {text: 'galeria', url: '/gallery'},
     {text: 'kontakt', url: '/contact'},
-];
\ No newline at end of file
+];
+
+const ulElement = document.createElement('ul');
+
+for(let i=0; i<3; i++) {
```

Comment:
Tutaj musi być uniwersalna rozwiązanie. Jak ktoś doda kolejny element do tablicy to pewnie nie będzie pamietał, aby zmienić wartość w pętli. Dlatego lepiej jest zrobić tak: `for(let i=0; i<menuItems.length; i++) {`

Reply:
Faktycznie